nekoyume/Assets/_Scripts/UI/Widget/ArenaBoard.cs

```diff
@@ -87,10 +87,11 @@ public async UniTaskVoid ShowAsync(bool ignoreShowAnimation = false)
             UpdateBillboard();
             UpdateScrolls();
 
-            // NOTE: If `_playerScroll.Data` does not contains player, fix below.
+            // NOTE: This code assumes that '_playerScroll.Data' contains local player
+            //       If `_playerScroll.Data` does not contains local player, fix below.
```

Comment:
It would be batter to be more specific.

```suggestion
            //       If `_playerScroll.Data` does not contains local player, change `2` in the line below to `1`.
```